test/e2e/lazy_activation_policy_test.go

```diff
@@ -269,15 +272,52 @@ var _ = ginkgo.Describe("Lazy activation policy testing", func() {
 			ginkgo.By("step1: deployment would not propagate when lazy policy created after deployment", func() {
 				// wait to distinguish whether the deployment will not propagate or have no time to propagate
 				time.Sleep(waitIntervalForLazyPolicyTest)
+				// when create lazy policy after resource, a suspended Binding will be created to prevent resource from dispatching until resource is updated by User.
+				framework.WaitResourceBindingFitWith(karmadaClient, namespace, bindingName, func(rb *workv1alpha2.ResourceBinding) bool {
+					return *rb.Spec.Suspension.Dispatching == true
+				})
```

Comment:
It should be enough for us to check this for now. You don't need to wait for a while and check the resources on the member clusters.